backend/store/queue.go

```
@@ -0,0 +1,162 @@
+
+func (ss QueueStore) InitQueue(userId uuid.UUID) error {
+	_, err := ss.db.Exec("insert into queue (user_id, next_position, current_position) values ($1, $2, $3) on conflict (user_id) do nothing", userId, 0, 0)
+	return err
+}
+
+func (ss QueueStore) ChangeCurrentSong(queueId uuid.UUID, songId uuid.UUID) error {
+	tx, err := ss.db.Beginx()
+	if err != nil {
+		return err
+	}
+	defer tx.Rollback()
```

golangci-lint (1.62.2)

Error return value of `tx.Rollback` is not checked

(errcheck)